src/components/Footer.tsx

```diff
@@ -44,7 +44,7 @@ export const Footer: React.FC = () => (
       </Flex>
     </Flex>
     <Flex fontSize="lg" textAlign="center" flexWrap="wrap" columnGap={4}>
-      <Text>2023 Solidity Team</Text>
+      <Text>2024 Solidity Team</Text>
```

Comment:
```suggestion
      <Text>{new Date().getFullYear()} Solidity Team</Text>
```

@drewroberts Thanks for this! How about doing this to make it dynamic, keeping it always up to date?

Just did the same thing over on the Remix website repo https://github.com/ethereum/remix-website/pull/70/commits/c83f94fbf91205011c844006790981a0ea320549